lib/pymedphys/_experimental/pinnacle/pinnacle.py

```diff
@@ -38,6 +38,31 @@
 # SOFTWARE.
 
 
+"""A tool to export DICOM objects from raw Pinnacle data:
+
+.. WARNING::
+   The DICOM objects exported by this tool are not the same as the DICOM
+   objects exported from within the Pinnacle application. Certain DICOM
+   modalities correspond fairly well to the Pinnacle exported objects
+   for some versions of Pinnacle. Others are not working as expected or
+   have not been validated.
+
+   The first step when using this tool is to compare the output generated
+   to the 'ground truth' generated by your version(s) of Pinnacle. You can
+   then determine if this tool currently meets your export needs, or if
+   some adjustments need to be made. Please create an issue on the PyMedPhys
+   GitHub describing the circumstances where the tool is not working,
+   providing some sample raw Pinnacle data where possible. If you're up for
+   a challenge, clone the PyMedPhys repository and attempt to solve the
+   problem yourself! Don't be afraid to ask for help when contributing your
+   changes. The more developers we can encourage to contribute to this tool,
+   the sooner it will become stable and reliable.
+
+   Finally, note that while this tool can be extremely useful for exporting
+   large amounts of Pinnacle data to DICOM, we recommend that this should
+   only be used for research purposes and not clinically.
+"""
+
```

Comment:
So that we aren't repeating ourselves and creating two "sources of truth" for this warning is there a way we can copy it across, similar to the approach undergone in the following:

https://github.com/pymedphys/pymedphys/blob/8c55ae66c182f7e268f235322b498fc1ac34eccf/lib/pymedphys/metersetmap.py#L30

Reply:
That's a good idea. I'll look into that.

Reply:
looks like I can use the include directive, I've created a subfolder, messages, within docs for future similar situations.